### DISCUSSION TITLE
Update OpenCV version

### DISCUSSION THREAD

**Discussion by xcadenas-veridas** [PaddlePaddle/PaddleOCR#13134] [Ideas & Features]
Hi everyone!

I see that the new version of PaddleOCR is limited to version 4.6.0.66 of OpenCV. Considering that version 4.10.0.84 of OpenCV is available, has an update been considered?

Also, do you have a Slack community where we can discuss questions more efficiently?

Thanks!!

**Comment by GreatV** [comment]
We are working on it. First of all, we need to remove the imgaug library.

**Reply by GreatV** [reply]
we have https://discord.gg/g9wMzW25.

**Comment by CapturefastCEO** [comment]
I updated the newest OpenCV version after PaddleOCR was installed, and it is working.

**Reply by GreatV** [reply]
@CapturefastCEO  Have you updated your NumPy version to 2.0?
